bouncer/util.py

```diff
@@ -70,4 +88,4 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return (annotation_id, document_uri, quote, text)
```

Comment:
I think the return value is getting a bit out of hand at this point. I'd advocate using either a [`namedtuple`](https://docs.python.org/3/library/collections.html#collections.namedtuple) or a map with named keys.